manifests/init.pp

```diff
@@ -113,9 +113,9 @@
 #   Similar to [`server`](#server), it can be an array of pools or a hash of pools to their respective options.
 #   See [pool](https://chrony.tuxfamily.org/doc/3.4/chrony.conf.html#pool)
 # @param refclocks
-#   This should be a Hash of hardware reference clock drivers to use.  They hash
-#   can either list a single list of options for the driver, or any array of
-#   multiple options if the same driver is used for multiple hardware clocks.
+#   This selects reference clock drivers to use. It can be an array of drivers, or a hash
+#   of drivers. The hash can either list a single list of options for the driver, or any
+#   array of multiple options if the same driver is used for multiple hardware clocks.
```

Comment:
This reads really confusing… can you show an `@example` of this?

Reply:
The example is still there, right below that description. I just added the mention that
Array also is valid. Perhaps we should not allow Arrays any more? But that might break
some setup someone might be using?

Reply:
You can use @example so it gets formatted properly.

https://puppet.com/docs/puppet/7/puppet_strings.html#puppet_strings_reference-strings-tags